### PR TITLE
[WEB3-4] Document auth endpoints

### PR DESCRIPTION
**Pull request by marciammart**
# Description

This PR adds documentation to the authentication endpoints using Swagger.

Ticket: https://runtime-revolution.atlassian.net/browse/WEB3-4

## Type of change
New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
The documentation can be accessed via the `/api-docs` endpoint

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/node-app/src/swagger.json [hunk 1]
```diff
@@ -536,6 +536,86 @@
...
+                    },
+                    {
+                        "name": "accessToken",
+                        "in": "query",
+                        "required": true
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "User",
```
Comment (goncaloadolfo) [inline packages/node-app/src/swagger.json:558]:
```suggestion
                        "description": "User API access token",
```

File: packages/node-app/src/swagger.json [hunk 2]
```diff
@@ -536,6 +536,86 @@
...
+                }
+
+            }
+        },
+        "/me": {
+            "get": {
+                "tags": ["authentication"],
+                "description": "Get user",
+                "responses": {
+                    "201": {
```
Comment (goncaloadolfo) [inline packages/node-app/src/swagger.json:585]: This should be 200 I believe

File: packages/node-app/src/swagger.json [hunk 3]
```diff
@@ -536,6 +536,86 @@
...
+                        "description": "Logged in",
```
Comment (goncaloadolfo) [inline packages/node-app/src/swagger.json:586]: Just be in sync with the remaining documentation

```suggestion
                        "description": "OK",
```

File: packages/node-app/src/swagger.json (hunk 2 above)
Comment (marciammart) [reply packages/node-app/src/swagger.json:585]: No, the endpoint is returning a 201

File: packages/node-app/src/swagger.json (hunk 3 above)
Comment (marciammart) [reply packages/node-app/src/swagger.json:586]: OK is for a 200 status, this is 201 that also creates a user in the database if they don't exist there already so 'Logged in' is a more comprehensive description

File: packages/node-app/src/swagger.json (hunk 2 above)
Comment (goncaloadolfo) [reply packages/node-app/src/swagger.json:585]: This is relative to /me endpoint. This returns a 200 with the user object

File: packages/node-app/src/swagger.json (hunk 3 above)
Comment (goncaloadolfo) [reply packages/node-app/src/swagger.json:586]: Same as the previous comment, this is relative to /me endpoint, and not /login

File: packages/node-app/src/swagger.json [hunk 4]
```diff
@@ -0,0 +1,714 @@
...
+                      "required": true
+                    },
+                    {
+                        "name": "accessToken",
+                        "in": "query",
+                        "required": true
+                    }
+                ],
+                "responses": {
+                    "200": {
```
Comment (goncaloadolfo) [inline packages/node-app/src/swagger.json:557]: Here it should be 201

**Review by goncaloadolfo — APPROVED** [review]

**Pull request merged**